example-library/src/main/resources/application-example.yaml

```
@@ -13,7 +13,13 @@
 # limitations under the License.
 #
 
+# Not interpreted by any java code, simply used by reference in the population.resources section
+default:
+  rootId: ../Palisade-examples/resources/data/
+  connectionDetail: data-service
+
 population:
+  user: example
```

could this be? Just to make the config easier to understand
```suggestion
  userConfigProfile: example
```